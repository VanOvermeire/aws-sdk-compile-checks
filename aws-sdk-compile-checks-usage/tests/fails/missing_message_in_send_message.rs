use aws_config::BehaviorVersion;
use aws_sdk_compile_checks_macro::required_props;

#[required_props]
async fn do_call() {
    let aws_config = aws_config::load_defaults(BehaviorVersion::latest()).await;
    let _something = -3_i32.abs();
    let sqs_client = aws_sdk_sqs::Client::new(&aws_config);
    sqs_client.send_message()
        .queue_url("...")
        .send()
        .await
        .expect("Call to succeed");
}

fn main() {}
