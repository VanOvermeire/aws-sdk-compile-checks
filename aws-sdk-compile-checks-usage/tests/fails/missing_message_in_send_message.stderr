error: method `send_message` is missing required argument(s): `message_body`
 --> tests/fails/missing_message_in_send_message.rs:9:16
  |
9 |     sqs_client.send_message()
  |                ^^^^^^^^^^^^
